packages/medusa/src/repositories/money-amount.ts

```diff
@@ -119,18 +119,24 @@ export class MoneyAmountRepository extends Repository<MoneyAmount> {
 
   public async findManyForVariantInPriceList(
     variant_id: string,
-    price_list_id: string
+    price_list_id: string,
+    required: boolean = false
```

Comment:
suggestion: Make this prop more descriptive. Hard to understand, what it's used for at a first glance

Reply:
Yeah i am agree, was thinking of required because an innerjoin requires both side. What do you think of shouldInnerJoin ?

Reply:
I am fine with that, but it's still maybe a bit too technical. Wdyt of `priceListsRequired` or `requiresPriceList`?

Reply:
Quite nice indeed, the inly thing is that when passing the arguments we already think that the priceList is required. I've been trapped by that btw 😅 should We rename the method to stand that the priceLists are optional by default then?